app/logical/sources/strategies/deviant_art.rb

```diff
@@ -204,7 +223,7 @@ def dtext_artist_commentary_desc
       end
 
       def self.deviation_id_from_url(url)
-        if url =~ ASSET
+        if url =~ ASSET or url =~ FAVME
```

Comment:
There's not really a problem here but just so you know:
There is a difference in ruby between `||` and `or` (`&&` and `and`): the difference is precedence

Check these [SO](https://stackoverflow.com/a/2083118) [answers](https://stackoverflow.com/a/2083630) (apparently `and`/`or` are intended for control flow and not logic?)

Since rubocop complains, I guess it should be changed but it complains about so much other stuff prevalent in the code base - even `$~` - so I don't know...

Reply:
Ah, that slipped past rubocop because I had turned it off for a second to stop blinding me with the regex complaints. I'll fix it.

Reply:
`||` and `&&` are preferred in Danbooru code.